cli/command/system/prune.go

```diff
@@ -34,7 +34,7 @@ func NewPruneCommand(dockerCli *command.DockerCli) *cobra.Command {
 	flags := cmd.Flags()
 	flags.BoolVarP(&opts.force, "force", "f", false, "Do not prompt for confirmation")
 	flags.BoolVarP(&opts.all, "all", "a", false, "Remove all unused images not just dangling ones")
-	flags.Var(&opts.filter, "filter", "Provide filter values (e.g. 'until=<timestamp>')")
+	flags.Var(&opts.filter, "filter", "Provide filter values (e.g. 'label=<key>=<value>')")
```

Comment:
Oh! I did't know about this!
I'll update the PR then, it's much simpler this way.